lib/response.js

```diff
@@ -146,17 +157,17 @@ function gitlabActionProjects (req, res, note) {
       const ret = { baseURL: config.gitlab.baseURL, version: config.gitlab.version }
       ret.accesstoken = user.accessToken
       ret.profileid = user.profileid
-      request(
-        config.gitlab.baseURL + '/api/' + config.gitlab.version + '/projects?membership=yes&per_page=100&access_token=' + user.accessToken,
-        function (error, httpResponse, body) {
-          if (!error && httpResponse.statusCode === 200) {
-            ret.projects = JSON.parse(body)
-            return res.send(ret)
-          } else {
-            return res.send(ret)
-          }
+      const apiUrl = `${config.gitlab.baseURL}/api/${config.gitlab.version}/projects?membership=yes&per_page=100&access_token=${user.accessToken}`
+      fetch(apiUrl).then(resp => {
+        if (!resp.ok) {
+          res.send(ret)
+          throw new Error('HTTP request returned did not okay-ish status')
```

Comment:
```suggestion
          throw new Error('HTTP request returned not okay-ish status')
```

Reply:
Yep, don't know what happened there in my mind.

Reply:
Fixed